test-configs.yaml

```diff
@@ -458,6 +458,9 @@ device_types:
     mach: arc
     class: arc-dtb
     boot_method: uboot
+    filters:
+      blacklist:
```

Comment:
Missing `-` as this is a list of filters.

Reply:
OK, updated.